.githooks/pre-commit

```diff
@@ -45,6 +45,18 @@ else
     against=16bbb57
 fi
 
+[[ -z "$files" ]] && exit
+# Change the copyright date at the top of any text files
+for file in $files; do
+    [[ -L $file ]] && continue
+    echo "Processing copyright dates in $file"
+    if [[ -e $file ]]; then
+        /usr/bin/perl -pi -e 'INIT { exit 1 if !-f $ARGV[0] || -B $ARGV[0]; $year = (localtime)[5] + 1900 }
+            s/^([*\/#\/"*[:space:]]*)Copyright\s+(?:\(C\)\s*)?(\d+)(?:\s*-\s*\d+)?\s(Advanced\s*Micro\s*Devices)/qq($1Copyright (C) $2@{[$year != $2 ? "-$year" : ""]} $3)/ie
+            if $. < 10' "$file" && git add -u "$file"
```

Comment:
Will this add the files to the current commit? If copyrights are updated we want to have that as a seperate commit.